wiki/osu!tourney/Skinning/th.md

```diff
@@ -0,0 +1,21 @@
+# osu!tourney skinning
+
+![The client can be customized in various ways](img/Osutourneycustom.png)
+
+Client นั้นมีการรองรับการปรับแต่งต่าง ๆ เพื่อที่คุณจะได้ปรับแต่งมันให้เข้ากับทัวร์นาเมนต์ของคุณ ซึ่งคุณจะต้องสร้างโฟลเดอร์ `/Skins/User/tournament` ในที่ติดตั้งของ client การปรับแต่งรองรับรูปแบบไฟล์ `.jpg` และ `.png`
+
+สิ่งต่อไปนี้สามารถปรับแต่งได้:
+
+- `background` - พื้นหลังที่ถูกใช้โดย client พื้นหลังค่าเริ่มต้นสามารถหาได้[ที่นี่](https://s.ppy.sh/images/tournament/default.png).
+- `background-win1` (ไม่บังคับ) - พื้นหลังที่จะแสดงตอนทีมสีน้ำเงินชนะระหว่างผลลัพธ์กำลังแสดงอยู่
+- `background-win2` (ไม่บังคับ) - พื้นหลังที่จะแสดงตอนทีมสีแดงเงินชนะระหว่างผลลัพธ์กำลังแสดงอยู่
+- `tourney-title` (ไม่บังคับ) - รูปภาพที่แสดงข้างล่างของ client หรือ บนพื้นหลัง
+  - สามารถนำมาแสดงโลโก้ของทัวร์นาเมนต์ได้
+
+![Team icons can be displayed beside the team names](img/Osutourneyicons.png)
+
+Client จะแสดงไอค่อนที่อยู่ใน `/Skins/User/tournament/icons` ไว้ข้าง ๆ ชื่อทีม ซึ่งสามารถใช้แสดง ธงของประเทศ หรือ รูปโปรไฟล์
+
+ชื่อของไอค่อนต้องตรงกับชื่อของทีม เช่นถ้าห้องถูกเรียกว่า `Test Tourney: (Team 1) vs (Team 2)` ไอค่อนต้องมีชื่อว่า `Team 1.jpg` และ `Team 2.jpg` ไอค่อนสามารถเป็น `.jpg` หรือ `.png` ก็ได้ และความคมชัดควรเป็น 50x50px
```

Comment:
```suggestion
Client จะแสดงไอคอนที่อยู่ใน `/Skins/User/tournament/icons` ไว้ข้าง ๆ ชื่อทีม ซึ่งสามารถใช้แสดง ธงของประเทศ หรือ รูปโปรไฟล์

ชื่อของไอคอนต้องตรงกับชื่อของทีม เช่นถ้าห้องถูกเรียกว่า `Test Tourney: (Team 1) vs (Team 2)` ไอคอนต้องมีชื่อว่า `Team 1.jpg` และ `Team 2.jpg` ไอคอนสามารถเป็น `.jpg` หรือ `.png` ก็ได้ และความคมชัดควรเป็น 50x50px
```
In other TH wiki file I see that it use `ไอคอน` instead of `ไอค่อน` (And in Google too) so I suggest to change it.